tests/containers/agency/AgencyFooterContainer-test.jsx

```diff
@@ -10,33 +10,63 @@ import sinon from 'sinon';
 import { AgencyFooterContainer } from
     'containers/agency/AgencyFooterContainer';
 import Router from 'containers/router/Router';
+import * as AgencyHelper from 'helpers/agencyHelper';
 
-import { mockLoad } from './mockAgency';
-
-
-const routerPushSpy = sinon.spy(Router.history, 'push');
-const loadAgencySpy = sinon.spy(AgencyFooterContainer.prototype, 'loadAgency');
+import { mockLoad, mockAgencyApi } from './mockAgency';
 
 // force Jest to use native Node promises
 // see: https://facebook.github.io/jest/docs/troubleshooting.html#unresolved-promises
 global.Promise = require.requireActual('promise');
 
+
+const routerPushSpy = sinon.spy(Router.history, 'push');
+
+const mockAgencyHelper = (functionName, event, expectedResponse) => {
```

Comment:
We have stopped doing this type of helper mock several sprints ago. Please review how they are being mocked here: https://github.com/fedspendingtransparency/usaspending-website/blob/dev/tests/containers/agency/visualizations/FederalAccountContainer-test.jsx#L14